jormungandr-lib/src/interfaces/mod.rs

```diff
@@ -63,4 +63,5 @@ pub use self::utxo_info::{UTxOInfo, UTxOOutputInfo};
 pub use self::value::{Value, ValueDef};
 pub use self::vote::{
     Payload, Tally, TallyResult, VotePlanDef, VotePlanStatus, VoteProposalStatus,
+    MEMBER_PUBLIC_KEY_BECH32_HRP,
```

Comment:
I think this name is less ambiguous if namespaced under `interfaces::vote` without the reexports.

But then, the same applies to `Payload` and maybe `Tally`.

Reply:
Another style idea is to collect all HRPs under the same mod namespace structure with clear naming logic, like

```
crate::bech32::vote::MEMBER_PUBLIC_KEY_HRP
crate::bech32::vote::COMMUNICATION_PUBLIC_KEY_HRP
```

Reply:
Yeah, I like that.